mission_control/README.md

```diff
@@ -3,15 +3,9 @@ The `mission control` node provides an implementation of missions using behavior
 `Mission control` uses the [BehaviorTree.CPP](https://www.behaviortree.dev/) library for mission definition. A Behavior Tree (BT) __is a way to structure the switching between different tasks__. This is nothing more than a mechanism to invoke callbacks at the right time under the right conditions.
 
 ## Features
-- Topic interface for:
-    - Abort Mission.
-    - Execute Mission.
-    - Load Mission.
-    - Query Missions.
-    - Remove Missions.
-    - Report Execute Mission State.
-    - Report Load Mission State.
-    - Report Missions.
+- Missions can be designed using [Groot](https://github.com/BehaviorTree/Groot)
+- Missions are described as behavior trees in XML
+- Abort missions and go to surface in the event of unexpected anomalies (e.g. system faults)
```

Comment:
@patapufate hmm, did you mean something like:
```suggestion
- Safe mission aborts -- vehicle is commanded to surface in the event of unexpected anomalies (e.g. system faults)
```
?

Reply:
Fixed